src/io/output_time.cc

```diff
@@ -212,6 +256,9 @@ void OutputTime::write_time_frame()
 			// Remember the last time of writing to output stream
 			write_time = time;
 			current_step++;
+            
+            // unset validity of the output mesh after data are written at the current write time
+            is_output_mesh_valid_ = false;
```

Comment:
This flag is checked in make_output_mesh() which is called from field_set.output().
If the flag is not there, the output mesh would be constructed twice - once for flow, once for transport.
Since we have the same output mesh for both, I assumed we can do it once at the moment...